test/modules/ModuleLib/TestResults/EdgeHubRestartMessageResult.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.ModuleUtil.TestResults
+{
+    using System;
+    using System.Net;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Newtonsoft.Json;
+
+    public class EdgeHubRestartMessageResult : MessageTestResult
+    {
+        public EdgeHubRestartMessageResult(
+            string source,
+            DateTime createdAt,
+            string trackingId,
+            string batchId,
+            string sequenceNumber,
+            DateTime edgeHubRestartedTime,
+            DateTime messageCompletedTime,
+            HttpStatusCode messageCompletedStatusCode,
+            uint restartSequenceNumber)
```

Comment:
I think you can remove restartSequenceNumber if restartSequenceNumber and sequenceNumber are the same all the time.  btw, can we use uint for sequenceNumber.

Reply:
No. I'm not changing the seqNum to uint. Message uses long. I'm going to keep it as that.